docs/source/techniques/ISIS_SingleCrystalDiffraction_Workflow.rst

```diff
@@ -0,0 +1,438 @@
+                        'GetTOFWindowFromBackToBackParams': True, 'NFWHM': 10,
+                        'LorentzCorrection': True, 'ScaleThetaWidthByWavelength': True,
+                        'OptimiseXWindowSize':True, 'ThresholdIoverSigma': 80}
+
+    for run in range(42730, 42733):
+        wish.process_data([run])
+        # load UB
+        wish.load_isaw_ub(path.join(save_dir, f'{run}.mat'), tol=0.15, run=run)
+        # predict peaks
+        wish.predict_peaks(MinDSpacing=0.75, WavelengthMin=0.85,
+                           ReflectionCondition='Body centred', run=run)
+        # Integrate
+        skew_kwargs = {'OutputFile': path.join(save_dir, f'{run}_integrated.pdf'),
+                       **intPeaksSkewArgs}
+        wish.integrate_data(INTEGRATION_TYPE.SKEW, PEAK_TYPE.PREDICT, run=run, **skew_kwargs)
+        wish.save_peak_table(run, PEAK_TYPE.PREDICT, INTEGRATION_TYPE.SKEW,
+                             save_dir=save_dir, save_format="Jana")
+        wish.delete_run_data(run)  # delete raw workspace to save memory
+
+    wish.save_all_peaks(PEAK_TYPE.FOUND, INTEGRATION_TYPE.SKEW, save_dir=save_dir, save_format="jana")
+
+b. Q Integration (with satellites)
+==================================
+
+..  code-block:: python
+
+    from mantid.simpleapi import *
+    from os import path
+    from Diffraction.single_crystal.base_sx import PEAK_TYPE, INTEGRATION_TYPE
+    from Diffraction.wish.wishSX import WishSX
+
+    # Setup WishSX
+    ##############
+    wish = WishSX(vanadium_runno=43526)
+    # set sample material
+    sphere = '''<sphere id="sphere">
+                <centre x="0.0"  y="0.0" z="0.0" />
+                <radius val="0.0009"/>
+                </sphere>'''  # sphere radius 0.9mm
+    wish.set_sample(Geometry={'Shape': 'CSG', 'Value': sphere},
+                    Material={'ChemicalFormula': 'Ca3-Ga2-Ge3-O12', 'SampleNumberDensity': 0.001086})
+
+    # load vanadium
+    ###############
+
+    wish.process_vanadium()
+
+    # Load runs one at a time and integrate
+    #######################################
+
+    save_dir = "/babylon/Public/UserName"
+    # integration parameters
+    intPeaksMDArgs = {'ellipsoid': True, 'fixQAxis': True, 'fixMajorAxisLength': True, 'useCentroid': True, 'MaskEdgeTubes': False}
+
+    for run in range(42730, 42733):
+        wish.process_data([run])
+        # load UB
+        wish.load_isaw_ub(path.join(save_dir, f'{run}.mat'), tol=0.15, run=run)
+        # predict peaks
+        # main
+        wish.predict_peaks(MinDSpacing=0.75, WavelengthMin=0.85,
+                           ReflectionCondition='Body centred', run=run)
+        # satellite
+        wish.predict_peaks(peak_type=PEAK_TYPE.SATELLITE, run=run
+                           ModVector1="0.5,0,0", ModVector2="0,0.5,0", ModVector3="0,0,0.5",
+                           MaxOrder=1, CrossTerms=False, RequirePeaksOnDetector=True,
+                           ReflectionCondition='Body centred')
```

Comment:
I am getting this error at this line

`File "<unknown>", line 33
    ModVector1="0.5,0,0", ModVector2="0,0.5,0", ModVector3="0,0,0.5",
    ^
SyntaxError: invalid syntax`